main.tf

```diff
@@ -37,11 +37,23 @@ module "security_group_gitlab" {
   description         = "Security group for the gitlab EC2"
   name                = "${var.environment}-gitlab"
   vpc_id              = var.vpc_id
-  ingress_cidr_blocks = var.ingress_cidr_blocks
+  ingress_cidr_blocks = [var.ingress_cidr_blocks]
   ingress_rules       = ["https-443-tcp", "ssh-tcp", "openvpn-udp"]
+  ingress_with_cidr_blocks = [
+    {
+      from_port   = 2222
+      to_port     = 2222
+      protocol    = "tcp"
+      description = "New SSH"
```

Comment:
Probably "SSH port override for host OS" would me more descriptive

Reply:
Done. Changed to "SSH port override for host OS" would me more descriptive